src/devices/adurosmart.ts

```diff
@@ -122,6 +122,26 @@ const definitions: DefinitionWithExtend[] = [
         description: 'AduroSmart on/off relay',
         extend: [onOff({powerOnBehavior: false})],
     },
+    {
+        zigbeeModel: ['Smart Siren'],
```

Comment:
Can you use fingerprint with manufacturer name instead? The Zigbee model is quite generic. You can get the manufacturer name from the device about page

Example:
https://github.com/Koenkk/zigbee-herdsman-converters/blob/880047a5556e729de3f9245b332914ea79013134/src/devices/namron.ts#L337